packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -492,12 +493,18 @@ function useInitialFormValues({
       });
 
       const defaultUserValues = {
-        email: rescheduleUid
-          ? bookingData?.attendees[0].email
-          : parsedQuery["email"] || session.data?.user?.email || "",
-        name: rescheduleUid
-          ? bookingData?.attendees[0].name
-          : parsedQuery["name"] || session.data?.user?.name || "",
+        email:
+          rescheduleUid && bookingData && bookingData.attendees.length > 0
+            ? bookingData?.attendees[0].email
+            : !!parsedQuery["email"]
+            ? parsedQuery["email"]
+            : session.data?.user?.email ?? "",
```

Comment:
added scenario that owner rescheduling and we displaying their email on book form.